cmd/scaffold.go

```diff
@@ -868,6 +869,7 @@ func (fnb *FlowNodeBuilder) initDB() error {
 	opts := badger.
 		DefaultOptions(fnb.BaseConfig.datadir).
 		WithKeepL0InMemory(true).
+		WithCompression(options.Snappy). // turn on snappy compression
```

Comment:
There is no noticable difference comparing to using ZSTD, so will go with snappy